ghostinspector.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Deployer;
+
+desc('Run Ghostinspector test suite');
+task('ghostinspector:run', function () {
+    $config = get('ghostinspector', []);
+    if (!isset($config['version'])) {
+        $config['version' ] = 'v1';
+    }
+
+    if (empty($config['apikey']) || empty($config['testsuite']) || empty($config['version'])) {
```

Comment:
This should fail with an error instead. If someone uses this recipe, he can be expected to fill the required credentials.